styleguide/_themes/derek/scss/globalElements/footer.scss

```diff
@@ -18,9 +18,13 @@
   font-size: 1.2rem;
   text-decoration: none;
 
+  &:focus {
+    text-decoration: none;
+  }
+
+  &:focus,
   &:hover {
     color: $gray-light;
```

Comment:
unfortunately drupal is still setting `a:focus` for these.  This will need to get updated in bootstrap overrides as well. 
![image](https://cloud.githubusercontent.com/assets/5263371/18292936/a2e3fae0-7455-11e6-994a-8a229065174f.png)